backend/api/Database/Models/Report.cs

```diff
@@ -12,7 +12,7 @@ public class Report
         public string Id { get; set; }
 
         [Required]
-        public string RobotId { get; set; }
+        public Robot Robot { get; set; }
```

Comment:
Why was this changed back to Robot? Are we certain this behaves as expected? 
For example, when GETing a report object through the API, the Robot field will now always be null.